src/init.cpp

```diff
@@ -528,6 +528,22 @@ std::string LicenseInfo()
            "\n";
 }
 
+std::string ConfigHelp(const std::string& strBitcoinClient)
+{
+    return strprintf(_("All command-line options (except for '%s') may be specified in a configuration file, and all configuration file options may also be specified on the command line. "
+                       "Command-line options override values set in the configuration file."),
+               "-conf") +
```

Comment:
inline `-conf`?

Reply:
This is done on purpose to not translate a command line option.

Reply:
Ah nice!